assets/sections/manipulate-company-data-api-filing/templates/guides/overview.md

```diff
@@ -133,6 +135,10 @@ For Insolvency data change submissions made using the Transactions and Insolvenc
 
 - Postcode of the first insolvency practitioner associated with the case matches one of the postcodes of a Companies House office ("CF143UZ", "BT28BG", "SW1H9EX" or "EH39FF")
 
+For REA data change submissions made using the Transactions and REA API’s the following scenarios:
```

Comment:
API's here should be APIs, and should have a comma after it (typos in requested text, not implementation!)

Reply:
done